src/DynamoPublish/Models/PublishModel.cs

```diff
@@ -222,13 +276,13 @@ internal void Authenticate()
             }
         }
 
-        internal void SendAsynchronously(IEnumerable<IWorkspaceModel> workspaces, WorkspaceProperties workspaceProperties = null)
```

Comment:
It was an error to pass all the workspaces to this method. We now isolate out the `HomeWorkspaceModel` and pass it directly.